robot/basestation/static/js/arm.js

```diff
@@ -182,14 +187,18 @@ $(document).ready(function () {
     event.preventDefault()
     num = this.id[1]
     isOpen = !$(this.id).is(':checked')
-    armReq = function(msgs) {
-        if(msgs[0]) {
-          $(this.id)[0].checked = !isOpen
-        } else {
-          $(this.id)[0].checked = isOpen
-        }
+    armReq = function (msgs) {
+      if (msgs[0]) {
+        $(this.id)[0].checked = !isOpen
+      } else {
+        $(this.id)[0].checked = isOpen
+      }
     }
-    sendRequest("Arm", 'motor ' + num + ' loop ' + (isOpen) ? 'open' : 'closed', armReq)
+    sendRequest(
+      'Arm',
+      'motor ' + num + ' loop ' + isOpen ? 'open' : 'closed',
```

Comment:
```
num = 5
isOpen = true
'motor ' + num + ' loop ' + isOpen ? 'open' : 'closed'
```
expected result:
`'motor 5 loop open'`
actual result:
`'open'`

Reply:
```
> 'motor ' + num + ' loop ' + (isOpen) ? 'open' : 'closed'
< "open"
```
Was this problem ever really solved though?
Enclosing parentheses around the entire ternary statement (not just the boolean condition) seems to do the trick though.
```
> 'motor ' + num + ' loop ' + (isOpen ? 'open' : 'closed')
< "motor 5 loop open"
```

Reply:
Fixed